client/src/DbtLanguageClientManager.ts

```diff
@@ -93,7 +94,10 @@ export class DbtLanguageClientManager {
       try {
         await workspace.fs.stat(currentUri.with({ path: `${currentUri.path}/${DBT_PROJECT_YML}` }));
         const oneLevelUpPath = Uri.joinPath(currentUri, '..').path;
-        if (!ExtensionClient.DEFAULT_PACKAGES_PATHS.some(p => oneLevelUpPath.endsWith(p))) {
+        if (
+          !ExtensionClient.DEFAULT_PACKAGES_PATHS.some(p => oneLevelUpPath.toLocaleLowerCase().endsWith(p)) &&
+          !currentUri.fsPath.toLocaleLowerCase().endsWith('integration_tests')
```

Comment:
It is a bit not clear what is 'integration_tests'? Could you please move it to const with specific name?